Lib/test/test_capi/test_file.py

```diff
@@ -243,11 +269,21 @@ def test_file_write_to_ascii_file(self):
             )
 
     def test_file_write_invalid(self):
-        self.assertRaises(TypeError, self.write, object(), io.BytesIO(), 0)
-        self.assertRaises(AttributeError, self.write, object(), object(), 0)
-        self.assertRaises(TypeError, self.write, object(), NULL, 0)
-        self.assertRaises(AttributeError, self.write, NULL, object(), 0)
-        self.assertRaises(TypeError, self.write, NULL, NULL, 0)
+        wr = self.write
+        self.assertRaises(TypeError, wr, object(), io.BytesIO(), 0)
```

Comment:
Use a string instead of `object()`. It will be clearer what you write and why this fails.